setup.cfg

```diff
@@ -6,19 +6,19 @@ inline-quotes = single
 [isort]
 balanced_wrapping = True
 default_section = THIRDPARTY
-known_first_party = change, these, modules
+known_first_party = {{ project_name }}
```

Comment:
We should still indicate that more than just the project name will need to be put here. Like `courses`, `plan`, etc in PCX

Reply:
i think that can be specified in the README, considering that when `startproject` is called there's no first party apps, would be weird if we make people put in more modules to lint it that might not be created yet.

Reply:
Putting it somewhere in the README makes sense to me.